ios/RNStartTime.m

```diff
@@ -0,0 +1,33 @@
+#import <Foundation/Foundation.h>
+
+#import "RNStartTime.h"
+#import <sys/sysctl.h>
+#include <math.h>
+#import <QuartzCore/QuartzCore.h>
+
+@implementation RNStartTime
+
+RCT_EXPORT_MODULE()
+
+static CFTimeInterval processStartTime() {
+  size_t len = 4;
+  int mib[len];
+  struct kinfo_proc kp;
+  
+  sysctlnametomib("kern.proc.pid", mib, &len);
+  mib[3] = getpid();
+  len = sizeof(kp);
+  sysctl(mib, 4, &kp, &len, NULL, 0);
+  
+  struct timeval startTime = kp.kp_proc.p_un.__p_starttime;
+  return startTime.tv_sec + startTime.tv_usec / 1e6;
+}
+
+- (NSDictionary *)constantsToExport
+{
+  // processStartTime returns value in seconds
+  // we are converting it to milliseconds and cutting the floating point part
+  return @{ @"START_TIME": [NSNumber numberWithDouble:ceil(processStartTime() * 1000)] };
```

Comment:
why `ceil` and not `floor`? To not run into having 0ms operations?

Reply:
There won't be any 0ms edge cases since this is a UNIX Epoch Timestamp
And since this is the app start time, it might be better to use floor, since it will make the general startup time seem longer by at most 1ms and won't cut anything from the duration.